package.json

```diff
@@ -4,8 +4,7 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "test": "PORT=4321 NODE_ENV=test LOG_LEVEL=silent mocha ./src/**/*.test.js ./test/server/*.test.js --require test/babel-register --bail",
-    "test:integration": "NODE_ENV=test mocha ./test/integration/**/*.test.js --bail"
+    "test": "PORT=4321 NODE_ENV=test LOG_LEVEL=silent mocha ./src/**/*.test.js ./test/**/*.test.js --require test/babel-register --bail"
```

Comment:
Add `--timeout 5000` here to save the duplication

Reply:
I don't want all the tests to have a timeout of 5000, would be good to realise if rendering a component goes over 2 seconds for example.

Reply:
Although if that does happen I would probably give up.

Reply:
Ah yes, I only added it to the integration tests.